api/php/services/RestaurantService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+include "php/Dbgateways/RestaurantDbGateway.php";
+
+class RestaurantService {
+    private $restaurantdbGateway;
+
+    public function __construct() {
+        $this->restaurantDbGateway = new RestaurantDbGateway();
+    }
+
+    public function processReview (string $restaurantName, string $restaurantAddress, string $restaurantPhone,
+        string $restaurantWebsite, string $review, int $deliciousnessScore, int $serviceScore, int $experienceScore, int $pricingScore, float $pricingValue,
+        array $images) {
+        if (strlen($review) == 0) {
+            return "Review cannot be blank";
```

Comment:
Lines 15-16: This user input validation should be performed in the front-end. We don't want to send the data over to server side that is not valid. This can slow the server down since then it will have to make multiple requests if the review is blank.